src/include/sof/lib/memory_pm.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Jakub Dabek <jakub.dabek@intel.com>
+ */
+
+#ifndef __SOF_LIB_MEMORY_PM_
+#define __SOF_LIB_MEMORY_PM_
+
+#include <sof\lib\shim.h>
+
+#define EBB_SEGMENT_SIZE_ZERO_BASE = (EBB_SEGMENT_SIZE - 1
```

Comment:
Either you are missing something here or you are trying to use macros to glue some parts of code what should be avoided.